src/lib/graph-visualization-helpers.js

```diff
@@ -152,3 +152,22 @@ export const getMinimalSpanningTree = (links, acs = true) => {
 
   return mstLinks;
 };
+
+//  the function which returns metric for given collection and name of the vector
+
+export const getMetrics = async (qdrantClient, { collectionName }) => {
```

Comment:
there are no vector name in the arguments of the function

Reply:
fixed